frontend/src/data/ImagesClient.ts

```diff
@@ -0,0 +1,9 @@
+import Image from '../models/Image';
+
+import { IMAGE_API } from './ApiConstants';
+
+async function getImages(): Promise<Array<Image>> {
+  return fetch(IMAGE_API).then((res) => res.json());
+}
+
+export { getImages };
```

Comment:
As a BE dev, this was the only reasonable solution I could come up with: writing a client.

Reply:
This is perfect :smile: